docs/plugin-syntax-dynamic-import.md

```diff
@@ -4,6 +4,17 @@ title: @babel/plugin-syntax-dynamic-import
 sidebar_label: syntax-dynamic-import
 ---
 
+`@babel/plugin-syntax-dynamic-import` is needed to enable parsing for import().
+
+In Babel 7.8.0 it's enabled by default, so this plugin shouldn't be needed if you are using `@babel/core@>7.8.0`.
```

Comment:
Since this change line is needed starting from the v7.8.0 docs, could you create a `website/versioned_docs/version-7.8.0/plugin-syntax-dynamic-import.md` file copying this one? (Change the document id on line 2 in the new file matching `website/versioned_docs/version-7.4.0/plugin-syntax-dynamic-import.md`'s format).

Reply:
Donezo 👍